README.md

```diff
@@ -3,6 +3,21 @@
 [![Build Status](https://api.travis-ci.com/maxmind/GeoIP2-node.svg?branch=master)](https://travis-ci.com/maxmind/GeoIP2-node)
 [![Known Vulnerabilities](https://snyk.io/test/github/maxmind/GeoIP2-node/badge.svg)](https://snyk.io/test/github/maxmind/GeoIP2-node)
 
+## Database Reader
+
+### City Example
+
+```
+const reader = require('GeoIP2-node').Reader;
+
+Reader.open('/usr/local/share/GeoIP/GeoIP2-City.mmdb').then(reader => {
+  const response = reader.city('128.101.101.101');
+
+  console.log(response.country.isoCode) // 'US'
```

Comment:
Is this safe to do without checking if `response.country` is defined? In the other high level readers, we provide empty objects for this reason.

Reply:
Ah, not it isn't.  I'll make that change.